packages/xrpl/test/fixtures/rippled/accountLines.js

```diff
@@ -1,18 +1,24 @@
-'use strict';
-const _ = require('lodash');
-const BASE_LEDGER_INDEX = 8819951;
+const _ = require('lodash')
```

Comment:
Can we be more specific on this import to help with lodash slicing?

Reply:
@JST5000 This will get fixed when this file is updated to typescript

Reply:
Done